index.js

```diff
@@ -5028,7 +5082,7 @@ module.exports = {
                 additionalProperties: false,
                 properties: {
                     'q-applicant-physical-injuries': {
-                        title: 'Select any injuries to the skin on your head, face or neck',
+                        title: 'Select any injuries to the skin on the head, face or neck',
```

Comment:
are these to avoid having to translate I take it?

Reply:
Yes. Obviously the option to contextualise them was there, Julie felt it wasn't worth it and making this more generic was a better approach.